### PR TITLE
Basic classes outline

### PR DESCRIPTION
**Pull request by lksobczyk**

### CODE REVIEW

**Review by Tomonike — APPROVED** [review]
It's look good.

File: shm/Game.hpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class Game {
+public:
+    Game(size_t money, size_t days, size_t final_goal);
+    void StartGame();
+
+private:
+    size_t _money;
+    size_t _days;
+    size_t _final_goal;
```
Comment (Nessy90) [inline shm/Game.hpp]: maybe let's stick to camel case :)

File: shm/Time.cpp [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#include "Time.hpp"
+
+#include <algorithm>
+
+Time& Time::operator++() {
+    ++_time_elapsed;
```
Comment (Nessy90) [inline shm/Time.cpp]: camel case :)

File: shm/main.cpp [hunk 3]
```diff
@@ -1,19 +1,14 @@
...
-int main() {
-    Ship ship1;
-    Ship ship2(10, 100, 500);
-    Ship ship3(10, "Ala", 200, 600, 300);
-
-    ship1 += 15;
-    ship2 -= 40;
+constexpr size_t start_money = 1'000;
+constexpr size_t game_days = 100;
+constexpr size_t final_goal = 2'000;
```
Comment (Nessy90) [inline shm/main.cpp]: camel case

**Review by Nessy90 — CHANGES_REQUESTED** [review]

**Review by Nessy90 — APPROVED** [review]

**Pull request merged**